src/components/Dashboard/Pay/index.tsx

```diff
@@ -55,9 +55,14 @@ export default function Pay() {
     const payButtonText = metadata.payButton?.length
       ? metadata.payButton
       : t`Pay`
+
+    // v1 projects who still use 100% RR to disable pay
     const isV1AndMaxRR =
       terminal?.version === '1' && fcMetadata.reservedRate === 200
 
+    // Edge case for MoonDAO, upgraded to v1.1 but can't use payIsPaused for now
+    const isMoonAndMaxRR = projectId?.eq(199) && fcMetadata.reservedRate === 200
```

Comment:
**suggestion**: can we have `199` in a constant so it's less magic?

Reply:
Yeah good idea, got it